analyzers/tests/SonarAnalyzer.UnitTest/Metrics/CSharpExecutableLinesMetricTest.cs

```diff
@@ -826,6 +847,23 @@ public static bool IsLetter(char c)
                  ",
                 7, 12, 14);
 
+        [TestMethod]
+        public void MultiLineInvocation() =>
+            AssertLinesOfCode(
+                @"
+                        using System;
+                        using System.Collections.Generic;
+
+                            public static bool Foo(int a, int b)
+                            {
+                               return Foo(1,
```

Comment:
basically, this means we don't respect the existing docs.

Reply:
Yes, I would merge this PR if you agree and maybe tell Tom tomorrow to take a look on the issue to get some feedback about priorities.